src/microbe_stage/editor/BarHelper.cs

```diff
@@ -51,7 +51,7 @@ public static Color GetBarColour(SegmentedBar.Type type, string name, bool produ
                     case "baseMovement":
                         return GD.Load<Texture>("res://assets/textures/gui/bevel/baseMovementIcon.png");
                     case "osmoregulation":
-                        return GD.Load<Texture>("res://assets/textures/gui/bevel/osmoIcon.png");
+                        return GD.Load<Texture>("res://assets/textures/gui/bevel/osmoregulationIcon.png");
```

Comment:
The file was added with the incorrect case, I see so that's why it fails to load. You should rename the file to "osmoregulationIcon.png" (on windows you probably need to delete the file and then re-make it in a different folder with the right name and then move it back to the bevel folder).

Reply:
Also I'll just pre-emptively say that if the icon shows up with colours on the ATP balance bar, then that's inconsistent and a change needs to be done to ensure the icon is grayscale like the other ATP balance icons when it shows up in the balance bar.